docs/orleans/whats-new-in-orleans.md

```diff
@@ -467,21 +504,21 @@ builder.AddMemoryStreams<DefaultMemoryMessageBodySerializer>(
     });
 ```
 
-### Open Telemetry
+## Open Telemetry
 
 The telemetry system has been updated in Orleans 7.0 and the previous system has been removed in favor of standardized .NET APIs such as .NET Metrics for metrics and ActivitySource for tracing.
 
 As a part of this, the existing `Microsoft.Orleans.TelemetryConsumers.*` packages have been removed. We are considering introducing a new set of packages to streamline the process of integrating the metrics emitted by Orleans into your monitoring solution of choice. As always, feedback and contributions are welcome.
```

Comment:
We should include a bit here about the replacement. For counters, the dotnet-counters tool can be used to monitor them: `dotnet counters monitor -n MyApp --counters Microsoft.Orleans`. Similarly, OpenTelemetry metrics can add the `Microsoft.Orleans` meters. For distributed tracing, OpenTelemetry can be used to monitor `Microsoft.Orleans.Runtime` and `Microsoft.Orleans.Application` as in the .NET Conf video: https://youtu.be/WUZw78P-Zg8?t=1382 and in the corresponding [sample code here](https://github.com/ReubenBond/orleans/blob/ad86ca49688769e7c2f64a748f60611691f889b6/samples/GPSTracker/GPSTracker.Service/Program.cs#L14):

```csharp
builder.Services.AddOpenTelemetryMetrics(metrics =>
{
    metrics
        .AddPrometheusExporter()
        .AddMeter("Microsoft.Orleans");

    // Uncomment this to export metrics to Azure Monitor
    //metrics.AddAzureMonitorMetricExporter(config => config.ConnectionString = Environment.GetEnvironmentVariable("AZMONCS"))
});
```

Distributed Tracing can be enabled by configuring Open Telemetry:
```csharp
builder.Services.AddOpenTelemetryTracing(tracing =>
{
    // Set a service name
    tracing.SetResourceBuilder(ResourceBuilder.CreateDefault().AddService(serviceName: "GPSTracker", serviceVersion: "1.0"));

    tracing.AddAspNetCoreInstrumentation();
    tracing.AddSource("Microsoft.Orleans.Runtime");
    tracing.AddSource("Microsoft.Orleans.Application");

    tracing.AddZipkinExporter(zipkin =>
    {
        zipkin.Endpoint = new Uri("http://localhost:9411/api/v2/spans");
    });
});
```
and the silo builder:
```csharp

builder.Host.UseOrleans((ctx, siloBuilder) =>
{
    // ... snip ...
    siloBuilder.AddActivityPropagation();
});
```

Reply:
Added in [59aae8f](https://github.com/dotnet/docs/pull/32582/commits/59aae8f25cc1fcb757e0f23eaa0305179e289e0a)